app/assets/stylesheets/nav.scss

```diff
@@ -15,10 +15,10 @@
   height: $topbar-height;
 }
 
-.droplink li {
-  background: transparent;
+.top-bar-section ul li a {
+  background: #FF851B !important;
```

Comment:
Color literals like `#FF851B` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#FF851B` should be written as `#ff851b`<br>!important should not be used